pkgs/by-name/rc/rclone-ui/package.nix

```diff
@@ -0,0 +1,86 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+
+  cargo-tauri,
+  nodejs,
+  npmHooks,
+  fetchNpmDeps,
+
+  pkg-config,
+  wrapGAppsHook3,
+
+  openssl,
+  libsoup_2_4,
+  webkitgtk_4_1,
+  glib-networking,
+  libappindicator,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "rclone-ui";
+  version = "1.0.4";
+
+  src = fetchFromGitHub {
+    owner = "rclone-ui";
+    repo = "rclone-ui";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-KTi/vCHiZVRAmQAiVXSWHCTTv1NnsvM5UZg8cpuFbRQ=";
+  };
+
+  npmDeps = fetchNpmDeps {
+    name = "${finalAttrs.pname}-${finalAttrs.version}-npm-deps";
+    inherit (finalAttrs) src;
+    forceGitDeps = true;
+    hash = "sha256-18QkqqYS1kGY701FbFBHLvr5WBkJzxFgR9VMnydeelY=";
+  };
+
+  cargoRoot = "src-tauri";
+  buildAndTestSubdir = finalAttrs.cargoRoot;
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-o21of2eS2KZtg1U1E6RwdaA8jGhEVzg7HkgOv1k5wxI=";
+
+  # Disable tauri bundle updater, can be removed when #389107 is merged
+  patches = [ ./remove_updater.patch ];
+
+  postPatch = ''
+    substituteInPlace src-tauri/Cargo.toml \
+       --replace-fail 'name = "app"' 'name = "${finalAttrs.pname}"'
+  '';
+
+  nativeBuildInputs = [
+    npmHooks.npmConfigHook
+    nodejs
+
+    cargo-tauri.hook
+
+    pkg-config
+    wrapGAppsHook3
+  ];
+
+  buildInputs = [
+    openssl
+    libsoup_2_4
+    webkitgtk_4_1
+    glib-networking
+    libappindicator
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/rclone-ui \
```

Comment:
The program is now wrapped three times.

```
rclone-ui                    .rclone-ui-wrapped           .rclone-ui-wrapped_          ..rclone-ui-wrapped-wrapped
```

please see https://github.com/NixOS/nixpkgs/blob/b799b8e7d816221336cf4d320cb382d22b5b7c44/doc/languages-frameworks/gnome.section.md#when-using-wrapgapps-hook-with-special-derivers-or-hooks-you-can-end-up-with-double-wrapped-binaries-ssec-gnome-common-issues-double-wrapped